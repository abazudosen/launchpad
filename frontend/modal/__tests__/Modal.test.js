/* @flow */

import React from 'react';
import Modal from '../Modal';
import renderer from 'react-test-renderer';

describe('Modal', () => {
  test('renders normal', () => {
    const component = renderer.create(
      <Modal title="Test" onRequestClose={() => {}}>
        <div>Test</div>
      </Modal>,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
