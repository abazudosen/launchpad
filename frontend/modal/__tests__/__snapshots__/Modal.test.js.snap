// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal renders normal 1`] = `<noscript />`;
